docs/guide/extending_ez_platform_ui.md

```diff
@@ -165,6 +167,250 @@ $menu->reorderChildren(
 );
 ```
 
+## Tabs
+
+Many elements of the back-office interface, such as System Info or Location View, are built using tabs.
+
+You can extend existing tab groups with new tabs, or create your own tab groups.
+
+### Adding a new tab group
+
+To create a custom tab group with additional logic you need to create it at the level of compiling the Symfony container, using a CompilerPass:
+
+``` php
+// src/AppBundle/DependencyInjection/Compiler/CustomTabGroupPass.php
+
+namespace AppBundle\DependencyInjection\Compiler;
+
+use EzSystems\EzPlatformAdminUi\Tab\TabGroup;
+use EzSystems\EzPlatformAdminUi\Tab\TabRegistry;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+class CustomTabGroupPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition(TabRegistry::class)) {
+            return;
+        }
+
+        $tabRegistry = $container->getDefinition(TabRegistry::class);
+        $tabGroupDefinition = new Definition(
+            TabGroup::class,  // or any class that extends TabGroup
+            ['custom-tab-group']
+        );
+        $tabRegistry->addMethodCall('addTabGroup', [$tabGroupDefinition]);
+    }
+}
+```
+
+A shorter way that you can use when no custom logic is required, is to add your own tab with the new group.
+If a tab's group is not defined, it will be created automatically.
+
+A new tab group can be rendered using the `ez_platform_tabs` Twig helper:
+
+``` jinja
+<div class="my-tabs">
+    {{ ez_platform_tabs('custom-tab-group') }}
```

Comment:
Does it have other parameters?
(see  https://github.com/ezsystems/ezplatform-admin-ui/blob/a7f9c83fe818775aeee90b960cc65f5ebe60e94c/src/bundle/Resources/views/dashboard/block/all.html.twig )

Reply:
Do we update the Twig Helper page list at the same time?